test/arbeidsuforhet/ArbeidsuforhetOppfyltTest.tsx

```diff
@@ -0,0 +1,111 @@
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import React from "react";
+import { queryClientWithMockData } from "../testQueryClient";
+import { ARBEIDSTAKER_DEFAULT } from "../../mock/common/mockConstants";
+import { render, screen } from "@testing-library/react";
+import { navEnhet } from "../dialogmote/testData";
+import { ValgtEnhetContext } from "@/context/ValgtEnhetContext";
+import { expect } from "chai";
+import { NotificationContext } from "@/context/notification/NotificationContext";
+import {
+  VurderingResponseDTO,
+  VurderingType,
+} from "@/data/arbeidsuforhet/arbeidsuforhetTypes";
+import { arbeidsuforhetQueryKeys } from "@/data/arbeidsuforhet/arbeidsuforhetQueryHooks";
+import { addWeeks } from "@/utils/datoUtils";
+import {
+  createArbeidsuforhetVurdering,
+  createForhandsvarsel,
+} from "./arbeidsuforhetTestData";
+import { ArbeidsuforhetOppfylt } from "@/sider/arbeidsuforhet/ArbeidsuforhetOppfylt";
+
+let queryClient: QueryClient;
+
+const mockArbeidsuforhetVurderinger = (vurderinger: VurderingResponseDTO[]) => {
+  queryClient.setQueryData(
+    arbeidsuforhetQueryKeys.arbeidsuforhet(ARBEIDSTAKER_DEFAULT.personIdent),
+    () => vurderinger
+  );
+};
+
+const renderArbeidsuforhetOppfyltSide = () => {
+  render(
+    <QueryClientProvider client={queryClient}>
+      <ValgtEnhetContext.Provider
+        value={{ valgtEnhet: navEnhet.id, setValgtEnhet: () => void 0 }}
+      >
+        <NotificationContext.Provider
```

Comment:
Tror du kan fjerne denne? Hvis vi ikke skal legge inn `NotificationsProvider` og ha notification-toast her.

Reply:
Ja, godt sett! 👀 Det ser ut til å funke fint uten toast når det ble en egen route for oppfylt 👍